src/wasm/wasm-binary.cpp

```diff
@@ -71,8 +71,9 @@ int32_t WasmBinaryWriter::writeU32LEBPlaceholder() {
   return ret;
 }
 
-void WasmBinaryWriter::writeResizableLimits(Address initial, Address maximum, bool hasMaximum) {
-  uint32_t flags = hasMaximum ? 1 : 0;
+void WasmBinaryWriter::writeResizableLimits(Address initial, Address maximum,
+                                            bool hasMaximum, bool shared) {
+  uint32_t flags = (uint32_t) hasMaximum | (uint32_t) shared << 1;
```

Comment:
@binji This uses the 2nd bit as the "shared" bit, i.e 
 0x03 is the flags value for shared.  https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md#spec-changes says it's 0x11 but I'm wondering if you meant 0b11?